src/DetailsView/components/left-nav/left-nav-link-builder.tsx

```diff
@@ -66,6 +69,10 @@ export function generateAssessmentTestKey(
 export type assessmentTestKeyGenerator = typeof generateAssessmentTestKey;
 
 export class LeftNavLinkBuilder {
+    public constructor(
```

Comment:
I think it may be easier/simpler to use the feature flag store data available in both assessment-left-nav and quick-assess-left-nav and pass it as a parameter or just simply check the state in the left nav components themselves.

This allows us to not have to manage if the state hasn't been loaded and what not here.